content/200-concepts/100-components/01-prisma-schema/08-features-without-psl-equivalent.mdx

```diff
@@ -44,15 +44,34 @@ There are two reasons to use a database-level function:
 
   The UUID is _only_ generated if you use Prisma Client to create the `Post`. If you create posts in any other way, such as a bulk import script written in plain SQL, you must generate the UUID yourself.
 
-### Enable extensions for native database functions
+### Enable PostgreSQL extensions for native database functions
 
-Some native database functions are part an extension. For example, `gen_random_uuid()` is part of the [`pgcrypto`](https://www.postgresql.org/docs/10/pgcrypto.html) module, and the `@db.Citext` native type attribute expects the `citext` extension. You must install the extension in order to use any functions or features:
+In PostgreSQL, some native database functions are part of an extension. For example, in PostgreSQL versions 12.13 and earlier, the `gen_random_uuid()` function is part of the [`pgcrypto`](https://www.postgresql.org/docs/10/pgcrypto.html) extension.
+
+To use a PostgreSQL extension, it must first be installed on the file system of your database server.
+
+In Prisma versions 4.5.0 and later, you can then activate the extension by declaring it in your Prisma schema with the [`postgresqlExtensions` preview feature](/concepts/components/prisma-schema/postgresql-extensions):
```

Comment:
Hm, I'd really like to emphasise the version, though, by having it at the start of the sentence.

Reply:
It depends on how hardline we want to be about avoiding gerunds. It can be very tricky sometimes. One other alternative I thought of when I was looking at this PR was to restructure it slightly, and put some headings in. But it's your call, of course.

Reply:
Thanks - I'm going to leave as is for now.